### PR TITLE
autossh: homepage inclusion

### PR DESCRIPTION
**Pull request by malfisya**
**Summary**

- Added `homepage` key to `package.yml` (Part of #411)

**Test Plan**

- Verified the added homepage loads information about the package

**Checklist**

- [x] Package was built and tested against unstable

**Comment**

Please put this [tarball](https://fossies.org/linux/privat/autossh-1.4g.tgz) on sources.getsol.us, their [source](https://www.harding.motd.ca/autossh/autossh-1.4g.tgz) doesn't work for me

### CODE REVIEW

**Review by TraceyC77 — DISMISSED** [review]
Looks good. Thanks!

File: packages/a/autossh/package.yml [hunk 1]
```diff
@@ -1,8 +1,9 @@
 name       : autossh
 version    : 1.4g
-release    : 2
+release    : 3
 source     :
-    - https://www.harding.motd.ca/autossh/autossh-1.4g.tgz : 5fc3cee3361ca1615af862364c480593171d0c54ec156de79fc421e31ae21277
+    - https://fossies.org/linux/privat/autossh-1.4g.tgz : 5fc3cee3361ca1615af862364c480593171d0c54ec156de79fc421e31ae21277
```
Comment (TraceyC77) [inline packages/a/autossh/package.yml:5]: Is there a reason you changed from the source tarball from the author's site to a 3rd party site? The original link works for me to download the tarball.

**Review by TraceyC77 — CHANGES_REQUESTED** [review]
I did get this to build, but I would rather keep the original source for the tarball unless there's a good reason to change it.

File: packages/a/autossh/package.yml (hunk 1 above)
Comment (malfisya) [reply packages/a/autossh/package.yml:5]: The original one doesnt work for me. I don't know if it is geoblock or something

Comment (TraceyC77) [reply packages/a/autossh/package.yml:5]: I'd like to investigate the issue of the file not working for you there before we change the package to use a 3rd party source. What happens when you try to load https://www.harding.motd.ca/autossh/autossh-1.4g.tgz in your browser?

What happens if you just go to https://www.harding.motd.ca/ ?

Comment (malfisya) [reply packages/a/autossh/package.yml:5]: > 
> The connection was reset
> 
> The network link was interrupted while negotiating a connection. Please try again.
> 
>     The site could be temporarily unavailable or too busy. Try again in a few moments.
>     If you are unable to load any pages, check your device’s data or Wi-Fi connection.
> 
> 

This is what happens. I have custom DNS (mullvad), disabling it doesn't help. Doesn't matter if this is Linux,  Windows, or Android Phone.

Comment (malfisya) [reply packages/a/autossh/package.yml:5]: Now, just tried with cellular  it worked ???? My router? Or ISP? Weird

**Comment by algent-al** [comment]
For this workaround `./configure $(sed 's|--runstatedir=/run||g' <<< "%CONFOPTS%")` there is a new macro available, `%configure_no_runstatedir`. Please use that.

**Pull request merged**